vr-wg-charter.html

```diff
@@ -269,7 +264,7 @@ <h2>
         <p>
           More detailed milestones and updated publication schedules are
           available on the <a href=
-          "https://www.w3.org/wiki/[groupname]/PubStatus">group publication
+          "https://www.w3.org/wiki/Webvr/PubStatus">group publication
```

Comment:
I suggest add `class="todo"` so this gets updated later.

(As a side comment, might be more practical to use an .md file in the spec repo for tracking publication status and other related stuff, but we'll figure that out later.)